src/electionguard/bignum.c

```diff
@@ -104,7 +104,9 @@ void Crypto_parameters_new()
 void Crypto_parameters_free()
 {
     mpz_clear(p);
+    mpz_clear(q);
```

Comment:
because of the way the API calls crypto_parameters_new and crypto_parameters_free on each call and in explicit context, these were leaking.  Is there a reason they need to be kept around?  I couldn't find any reference to them outside of the new/free scope cycle

Reply:
I think what you have here is right. `q` and `bignum_one` should be cleaned up within each new/free cycle since like the other global bignum variables.